readme.md

```diff
@@ -499,6 +499,19 @@ An instance of [`CacheableLookup`](https://github.com/szmarczak/cacheable-lookup
 **Note:** This should stay disabled when making requests to internal hostnames such as `localhost`, `database.local` etc.\
 `CacheableLookup` uses `dns.resolver4(..)` and `dns.resolver6(...)` under the hood and fall backs to `dns.lookup(...)` when the first two fail, which may lead to additional delay.
 
+###### dnsIpVersion
```

Comment:
```suggestion
###### lookupIpVersion
```

@sindresorhus Maybe this?

Reply:
Any news about this?

Reply:
I would go with `dnsLookupIpVersion`. It's verbose, but it's not a commonly-used option, so I think clarity is better.